js/core/core.internal.js

```diff
@@ -271,3 +271,10 @@ var _unique = function ( src )
 	return out;
 };
 
+// Array.isArray polyfill.
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray
+if (! Array.isArray) {
+    Array.isArray = function(arg) {
```

Comment:
Leaks into global space (if unset), is that OK? The rest of the functions in here are localised